### PR TITLE
Add WMS preview to fv_stations

### PR DESCRIPTION
**Pull request by jdk24**
Create DB view in schema `station_data` to expose WMS calls with 100m BBOX to external clients.
This would ideally include a `basemap` layer for reference and a dedicated style for the station layer.

### CODE REVIEW

File: db/update-files.d/015_create_wms_call_view.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/*
+Create view returning relative wms URLs including bbox and CQL filter on relevant station 
+*/
+
+CREATE OR REPLACE VIEW station_data.fv_wms_calls
+AS WITH envelope AS (
+         SELECT s.idpk_station AS idpk,
+            s.station_name,
+            s.station_code,
+            st_envelope(st_buffer(s.wkb_geometry, 100::double precision)) AS env
```
Comment (chrismayer) [inline db/update-files.d/015_create_wms_call_view.sql:12]: Maybe using a var for the distance (here 100), so it is easily adaptable and it does not look like a magic number.

**Review by chrismayer — COMMENTED** [review]
Thanks for this. I added 2 remarks you might want to have a look at.

File: db/update-files.d/015_create_wms_call_view.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
+\c sauber_data
+
+/*
+Create view returning relative wms URLs including bbox and CQL filter on relevant station 
+*/
+
+CREATE OR REPLACE VIEW station_data.fv_wms_calls
```
[the remark this answers is not in the document]
Comment (jdk24) [reply db/update-files.d/015_create_wms_call_view.sql:7]: Good point. Will do.

File: db/update-files.d/015_create_wms_call_view.sql (hunk 1 above)
Comment (jdk24) [reply db/update-files.d/015_create_wms_call_view.sql:12]: Also a good point, but with the limitations of a Postgres view, I could do:

```
CREATE OR REPLACE VIEW station_data.fv_wms_calls AS 
>>>     WITH buffer AS ( 
>>> 		SELECT 100::double precision AS buf_size
>>>	), 
        envelope AS (
	        SELECT s.idpk_station AS idpk,
	            s.station_name,
	            s.station_code,
>>>            st_envelope(st_buffer(s.wkb_geometry, buffer.buf_size)) AS env
	        FROM station_data.tab_prediction p
	        JOIN station_data.lut_station s ON p.fk_station = s.idpk_station
>>>        CROSS JOIN buffer
>>>        GROUP BY s.idpk_station, s.station_name, s.station_code, s.wkb_geometry, buffer.buf_size
        ), bbox AS (
	        SELECT envelope.idpk,
	            envelope.station_code,
				envelope.station_name,
	        	concat(round(st_xmin(envelope.env::box3d)), ',', round(st_ymin(envelope.env::box3d)), ',', round(st_xmax(envelope.env::box3d)), ',', round(st_ymax(envelope.env::box3d))) AS string
	        FROM envelope
        )
 SELECT row_number() OVER () AS obj_id,
    bbox.idpk AS station_id,
    bbox.station_code,
    bbox.station_name,
    concat('geoserver/station_data/wms?service=WMS&version=1.1.0&request=GetMap&layers=basemap,fv_stations&bbox=', bbox.string, '&width=250&height=250&srs=EPSG%3A3035&styles=,fv_stations_wmscall&CQL_FILTER=INCLUDE;station_code=', quote_literal(bbox.station_code), '&FORMAT=image/png') AS wms_call
   FROM bbox
  ORDER BY bbox.idpk;
```
But that would complicate the query, not scale well, etc.

Maybe some comment like:

``` st_envelope(st_buffer(s.wkb_geometry, /*buffer size:*/ 100::double precision )) AS env ```
would do?

**Comment by jdk24** [comment]
The view is first created in the db update files, `008_create_station_view.sql`.

I added a `DROP IF EXISTS` to the file to prevent issues when one redeploys the stack with the "newer" `fv_stations` version.

While we're meddling with previous files, I could create `fv_stations` in that file to begin with. 

The view is not referenced in any following update file.

**Review by chrismayer — APPROVED** [review]
Thanks @jdk24 , LGTM. Feel free to merge...

**Pull request merged**